akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/UserLeftVoiceConfEvtMsgHdlr.scala

```diff
@@ -54,5 +54,7 @@ trait UserLeftVoiceConfEvtMsgHdlr {
     if (liveMeeting.props.meetingProp.isBreakout) {
       BreakoutHdlrHelpers.updateParentMeetingWithUsers(liveMeeting, eventBus)
     }
+
+    if (Users2x.numUsers(liveMeeting.users2x) == 0) meetingEndTime = System.currentTimeMillis()

```

Comment:
Are you sure this case is a meeting end? If you are alone in a meeting in full audio and you refresh the page (i.e. no audio during the switch) do you hit this line?

Reply:
The issue you mention is dealt with by the `meetingEndTime` being reset to 0 when the first user joins the meeting. So, if someone is alone in the meeting and refreshes the page the `meetingEndTime` will be set to the current time, but when the page finishes reloading they will rejoin the meeting and the `meetingEndTime` will be set to 0 again.